grafana-plugin/src/components/AlertTemplates/CommonAlertTemplatesForm.config.ts

```diff
@@ -46,7 +46,8 @@ export const commonTemplateForEdit: { [id: string]: TemplateForEdit } = {
   sms_title_template: {
     name: TemplateOptions.SMS.key,
     displayName: 'Sms title',
-    description: '',
+    description:
+      "Result of this template will be used as title of SMS message. Please don't include any urls to avoid SMS message being blocked by carriers.",
```

Comment:
It seems some countries also don't like when phone numbers are included:
![image](https://github.com/grafana/oncall/assets/9406895/ef3826c7-c78f-4901-ba0d-552e92d4b63c)


```suggestion
      "Result of this template will be used as title of SMS message. Please don't include any urls, or phone numbers, to avoid SMS message being blocked by carriers.",
```